### ISSUE TITLE
tulip

### ISSUE THREAD

**Issue by PalashDebsharma** [PalashDebsharma/Tuilp#1]
<!DOCTYPE html>
<html lang="en">

	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width , initial-scale=1.0">
		
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
		
		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
		<base href="C:\Documents and Settings\Administrator\My Documents\Downloads\"/>

		<style>
			body{
				position:relative;
			}

			.affix{
				width:100%;
				z-index : 20;
			}
			.affix ~ .container-fluid {
     				position: relative;
     			}

			.navbar{

				font-family : "Lucida Console", Monaco, monospace;
				margin-bottom:0px;
			}

			#FirstContainerDiv{
				background-image : url("tulip.jpg");
				height:700px;
				margin-bottom : 10px;
			}

			#SecondContainerDiv{
				background-color : #CCCC66;
				padding : 30px;
				margin-top:10px;
				margin-bottom:10px;
			}
			.panel .panel-default{

				z-index: 10;
			}

			.panel .panel-heading, .panel-footer{
				background-color : #FF6666;
				font-family : "Lucida Console", Monaco, monospace;
			}

			.panel-body{
				padding-top : 25px;
				padding-left: 55px;
			}
			

			@media (max-width:350px)
			{			
			 	#MyCarousel .item img{
					height: 200px;
					width : 100%;
				}
				.jumbotorn p{
					font-size: 10px;
				}
			}
			@media (max-width:768px)
			{			
			 	#MyCarousel .item img{
					height: 250px;
					width : 100%;
				}
				.jumbotorn p{
					font-size: 10px;
				}
			}
			@media (max-width:992px)
			{			
			 	#MyCarousel .item img{
					height: 400px;
					width : 100%;
				}
			}
			@media (max-width:1200px)
			{			
			 	#MyCarousel .item img{
					height: 500px;
					width : 100%;
				}
			}
			
			#ThirdContainerDiv{

				margin-top:10px;
				z-index : 10;
			}
			
		</style>
		
	</head>

	<body >

		<div class="jumbotorn" style="height:200px; background-color:black;padding-bottom:0px;">

			<p class="text-center" style="color:white; padding: 100px; font-family:monospace;font-size:20px">
			 Tulip is a beautiful flower. We all love them</p>
			data-spy="scroll" data-target=".navbar" data-offset="10"
		</div>
		
		
		
		
		<div  class="navbar navbar-inverse" data-spy="affix" data-offset-top="50">
				<div class="container-fluid">
					<div class="navbar-header">
						<button class="navbar-toggle" data-toggle="collapse" data-target="#MyNavBar">
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span></button>

						<a href="" class="navbar-brand"> TULIP</a>
					</div>
				
					<div class="collapse navbar-collapse" id="MyNavBar" >
					<ul class="nav navbar-nav navbar-right">
						<li><a href="#FirstContainerDiv"> HOME</a></li>
						<li><a href="#SecondContainerDiv"> PURCHASE</a></li>
						<li><a href="#ThirdContainerDiv"> GALLERY</a></li>
						<li><a href="#FourthContainerDiv"> GROW</a></li>
						<li><a href="#"><span class="glyphicon glyphicon-user"></span> USER</a></li>
						<li><a href="#"><span class="glyphicon glyphicon-log-in"></span> LOGIN</a></li>
					</ul>
					</div>
				</div>
			</div>
		
	
			<div class="container-fluid" id="FirstContainerDiv" >
			<div class="row" >

			
				
			</div>
			</div>

			<div class="container-fluid" id="SecondContainerDiv"  >
			<div class="row" >

						<div class="col-sm-3">

							<div class="panel panel-default">
								<div class="panel-heading">
									
									<p class="panel-title text-center"> TULIP</p>
								</div>
								<div class="panel-body">

									<img src="tulip 0.jpg" class="img-responsive">
								</div>
								<div class="panel-footer">
									<a href="#" role="button" class="btn btn-default btn-block">PURCHASE</a>
								</div>
								
							</div>
						</div>

						<div class="col-sm-3">
							<div class="panel panel-default">
								<div class="panel-heading">
									
									<p class="panel-title text-center"> TULIP</p>
								</div>
								<div class="panel-body">

									<img src="tulip 0.jpg" >
								</div>
								<div class="panel-footer">
									<a href="#" role="button" class="btn btn-default btn-block">PURCHASE</a>
								</div>
								
							</div>

						</div>

						<div class="col-sm-3">
							<div class="panel panel-default ">
								<div class="panel-heading">
									
									<p class="panel-title text-center"> TULIP</p>
								</div>
								<div class="panel-body">

									<img src="tulip 0.jpg" >
								</div>
								<div class="panel-footer">
									<a href="#" role="button" class="btn btn-default btn-block">PURCHASE</a>
								</div>
								
							</div>

						</div>

						<div class="col-sm-3">
							<div class="panel panel-default">
								<div class="panel-heading">
							
									<p class="panel-title text-center"> TULIP</p>
								</div>
								<div class="panel-body">

									<img src="tulip 0.jpg" >
								</div>
								<div class="panel-footer">
									<a href="#" role="button" class="btn btn-default btn-block">PURCHASE</a>
								</div>
								
							</div>

						</div>
				</div>
			</div>
				
				
				<div "id="ThirdContainerDiv">
	
				<div class="carousel slide" data-ride="carousel" id="MyCarousel">
					
					<ol class="carousel-indicators">
						<li data-target="#MyCarousel" data-slide-to="0" class="active" ></li>
						<li data-target="#MyCarousel" data-slide-to="1"></li>
						<li data-target="#MyCarousel" data-slide-to="2"></li>
						<li data-target="#MyCarousel" data-slide-to="3"></li>

					</ol>

					<div class="carousel-inner" role="listbox">
						<div class="item active">

							<img  src="tulip 1.jpg">
						</div>
						<div class=" item">
							<img  src="tulip 2.jpg">
						</div>
						<div class="item">
							<img  src="tulip 3.jpg">
						</div>
						<div class="item">
							<img  src="tulip 4.jpg">
						</div>
					</div>

					<a href="#MyCarousel" class="left carousel-control" data-slide="prev"><span class="glyphicon glyphicon-chevron-left"></span></a>
					<a href="#MyCarousel" class="right carousel-control" data-slide="next"><span class="glyphicon glyphicon-chevron-right"></span></a>
				</div>
				</div>

				
				
				<div class="well" id="FourthContainerDiv" style="height=200px;background-color:black">
				</div>

			

		</div>
	
	</body>

</html>
